src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/TypeMapping.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers
+{
+    public record TypeMapping(string OldName, string NewName)
+    {
+        public string SimpleName => OldName.LastIndexOf('.') < 0
```

Comment:
do we need a null check here? I keep forgetting where we're using nullable-reference types.

Reply:
Good point. Since this type is public, we should probably check for null. I'll fix it.